nupic/algorithms/anomaly.py

```diff
@@ -154,9 +161,33 @@ def compute(self, activeColumns, predictedColumns,
       else:
         score = 0.0
 
+    self._iteration += 1
+    if not self.isReady():
+      score = 0.5
```

Comment:
ok TP.burnIn is 2, which does not make sense for any stable learning. Should we bother raising the default to (300 as in likelihood?), or just ignore this idea?